README.md

```diff
@@ -87,6 +88,7 @@ Once you have a git clone or recursive archive, building is the
 same as almost every autotools project:
 
 ```
+git submodule update --init
```

Comment:
`autogen.sh` takes care of that for us, no? Or is there something subtle going on here?